modules/serverless-rn-1-32-0.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-32-0_{context}"]
+= Red Hat {ServerlessProductName} 1.32
+
+{ServerlessProductName} 1.32 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes.
+
+[id="new-features-1-32-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.11.
+* {ServerlessProductName} now uses Knative Eventing 1.11.
+* {ServerlessProductName} now uses Kourier 1.11.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.11.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.11.
+* The `kn func` CLI plug-in now uses `func` 1.13.
+
+* Serverless Logic, which is available as a Technology Preview (TP) feature, has been updated.
++
+See link:https://openshift-knative.github.io/docs/docs/latest/serverless-logic/about.html[the Serverless Logic documentation] for usage instructions.
+
+* You can configure the {FunctionsProductName} readiness and liveness probe settings for the user-container and the queue-proxy container.
+
+* {FunctionsProductName} now supports {pipelines-shortname} versions `1.10` and `1.11`. The older versions of {pipelines-shortname} are no longer compatible with {FunctionsProductName}. 
```

Comment:
Quick note: We tested Functions on cluster against 1.11, 1.12, 1.13 and 1.14 (latest) version of Pipelines. I would probably not mention 1.10 version here.

Reply:
Correction form above comment: tested as well against 1.10. So all version from 1.10 till 1.14 (latest) are supported.

Reply:
Thank you! I have made the changes!